From 48f9247576bf533cacd4ab1106411a53becd71f5 Mon Sep 17 00:00:00 2001
From: lijiayi <lijiayi@eswincomputing.com>
Date: Tue, 17 Jun 2025 10:09:59 +0800
Subject: [PATCH 564/652] fix:modify model name

Changelogs:
1.modify model name A1 to A2.
2.add factory to model name.

Signed-off-by: lijiayi <lijiayi@eswincomputing.com>
Reviewed-by: Joey Zheng <zhengyu@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts b/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts
index c0ae9e0f4518c..c31a7615cb6c8 100644
--- a/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts
+++ b/arch/riscv/boot/dts/eswin/vpu7702_pcie_factory.dts
@@ -39,7 +39,7 @@
 / {
 	#address-cells = <2>;
 	#size-cells = <2>;
-	model = "ESWIN EIC7702 VPU A1";
+	model = "ESWIN EIC7702 VPU FACTORY A2";
 	compatible = "sifive,hifive-unmatched-a00", "sifive,fu740-c000",
 		     "sifive,fu740", "eswin,eic7700";
 
-- 
2.51.0

